### DISCUSSION TITLE
Purchase of the TL280 Module

### DISCUSSION THREAD

**Discussion by taofineberg** [BrianHumlicek/DSC-TLink#4] [General]
Hi, 
I have a neo alarm and would like to start working on this. I dont have the TL280. 
I was looking at purchasing this one. It looks like a good price and seems to cover ethernet as well as others.  
https://www.ebay.com/itm/176216727465

Do you see any issues with this version using your application

**Comment by BrianHumlicek** [comment]
I believe that will work for this project.  I notice that the SW version is 5.21.  I know the current version is something like 5.6 and I think there is some functionality that requires 5.4 and over.  You may be able to update it with DLS, but I think it will probably be fine for the purposes of this project.

Brian
